pkgs/development/compilers/gmqcc/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  pkgs,
+  stdenv,
+  fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+    pname = "gmqcc";
+    version = "unstable-2021-07-09";
+
+    src = fetchFromGitHub {
+      owner = "graphitemaster";
+      repo = "gmqcc";
+      rev = "297eab9e5e2c9cc4f41201b68821593a5cf9a725";
+      sha256 = "1hl2qn7402ia03kjkblj4q4wfypxkil99sivsyk2vrnwwpdp4nzx";
+    };
+
+    installPhase = ''
+      runHook preInstall
+
+      mkdir -p $out/bin
+      install -m755 gmqcc $out/bin
+
+      runHook postInstall
+    '';
+
+    meta = with lib; {
+      description = "A modern QuakeC compiler";
```

Comment:
I would suggest the following order:
- first, `homepage`;
- then, short `description`;
- then, `longDescription`;
- then the other things in alphabetical ordering